pulse_actions/handlers/treeherder_resultset.py

```diff
@@ -54,9 +55,9 @@ def on_resultset_action_event(data, message, dry_run, stage=False):
         else:
             status = 'Dry-mode, no request sent'
     elif action == "trigger_all_talos_jobs":
-        trigger_all_talos_jobs(repo_name, revision, times, dry_run=dry_run)
+        trigger_all_talos_jobs(repo_name, revision, times, priority, dry_run=dry_run)
```

Comment:
Can you also fix how this is called?
repo_name=repo_name,
revision=revision,
and on?